work-practices/external-meetings/README.md

```diff
@@ -0,0 +1,19 @@
+# External meetings
+
+External meetings are typically scheduled and managed by one or more partners. All meetings are scheduled on our respective [Google Calendars](../remote-work#google-calendar).
+
+## Calendly
+
+DataMade uses [Calendly](https://calendly.com/) to schedule meetings with clients and other people outside DataMade. Calendly syncs directly with Google Calendar and removes a lot of the back & forth work typically spent finding available times to meet.
+
+Currently, only partners have Calendly accounts. These are the currently available Calendly links:
+
+* Derek: https://calendly.com/derek-eder
```

Comment:
since this is a public repo, not sure we should make these links available since they allow someone to book directly on our calendars